src/test/resources/features/resolve/rules_and_checkpoints_names_resolve.feature

```diff
@@ -357,12 +357,20 @@ Feature: Resolve name after 'rules.' and 'checkpoints.' to their corresponding d
     Given a snakemake project
     And a file "boo.smk" with text
     """
+    include: "zoo.smk"
+
     rule something:
       input: "data_file.db"
 
     rule NAME:
       input: "file.txt"
     """
+    And a file "zoo.smk" with text
+    """
+    rule zoo_rule: threads: 1
```

Comment:
It is better when a test checks a single idea/feature. Here we test several things together - resolve to module and to a file included in module.. Could you please split test in several independent tests that better reflect which case is being tested.